WordPressKit/WordPressKit/BlogJetpackSettingsServiceRemote.swift

```diff
@@ -0,0 +1,177 @@
+
+        guard let monitorEnabled = dictionary[Keys.monitorEnabledKey] as? Bool,
+            let blockMaliciousLoginAttempts = dictionary[Keys.blockMaliciousLoginAttemptsKey] as? Bool,
+            let whitelistedIPs = dictionary[Keys.whiteListedIPAddressesKey]?[Keys.whiteListedIPsLocalKey] as? Array<String>,
+            let ssoEnabled = dictionary[Keys.ssoEnabledKey] as? Bool,
+            let ssoMatchAccountsByEmail = dictionary[Keys.ssoMatchAccountsByEmailKey] as? Bool,
+            let ssoRequireTwoStepAuthentication = dictionary[Keys.ssoRequireTwoStepAuthenticationKey] as? Bool else {
+                throw ResponseError.decodingFailure
+        }
+
+        return RemoteBlogJetpackSettings(monitorEnabled: monitorEnabled,
+                                         blockMaliciousLoginAttempts: blockMaliciousLoginAttempts,
+                                         loginWhiteListedIPAddresses: Set(whitelistedIPs),
+                                         ssoEnabled: ssoEnabled,
+                                         ssoMatchAccountsByEmail: ssoMatchAccountsByEmail,
+                                         ssoRequireTwoStepAuthentication: ssoRequireTwoStepAuthentication)
+    }
+
+    func remoteJetpackMonitorSettingsFromDictionary(_ dictionary: [String: AnyObject]) throws -> RemoteBlogJetpackMonitorSettings {
+
+        guard let monitorEmailNotifications = dictionary[Keys.monitorEmailNotificationsKey] as? Bool,
+            let monitorPushNotifications = dictionary[Keys.monitorPushNotificationsKey] as? Bool else {
+                throw ResponseError.decodingFailure
+        }
+        
+        return RemoteBlogJetpackMonitorSettings(monitorEmailNotifications: monitorEmailNotifications,
+                                                monitorPushNotifications: monitorPushNotifications)
+    }
+
+    func dictionaryFromJetpackSettings(_ settings: RemoteBlogJetpackSettings) -> [String: AnyObject] {
+
+        return [Keys.monitorEnabledKey: settings.monitorEnabled as AnyObject,
+                Keys.blockMaliciousLoginAttemptsKey: settings.blockMaliciousLoginAttempts as AnyObject,
+                Keys.whiteListedIPAddressesKey: settings.loginWhiteListedIPAddresses as AnyObject,
+                Keys.ssoEnabledKey: settings.ssoEnabled as AnyObject,
+                Keys.ssoMatchAccountsByEmailKey: settings.ssoEnabled as AnyObject,
+                Keys.ssoRequireTwoStepAuthenticationKey: settings.ssoEnabled as AnyObject]
+
+    }
+
+    func dictionaryFromJetpackMonitorSettings(_ settings: RemoteBlogJetpackMonitorSettings) -> [String: AnyObject] {
+
+        return [Keys.monitorEmailNotificationsKey: settings.monitorEmailNotifications as AnyObject,
+                Keys.monitorPushNotificationsKey: settings.monitorPushNotifications as AnyObject]
+    }
+}
+
+public extension BlogJetpackSettingsServiceRemote {
+
+  //  public struct Keys Enum {
+    public enum Keys {
```

Comment:
Sorry I missed this in the first round, but if we already have a `Keys` namespace, maybe we can drop the `Key` suffix from all constants?

Also I think `Key` is better than `Keys`, but I seem to be a minority there 😢 
```
$ ack '(enum|struct) Keys?\b' | grep -Eo 'Keys?' | sort | uniq -c
   3 Key
  11 Keys
```

Reply:
Sure thing, I that's probably a left over from when they were not an enum.